src/protocols/ethereum/views/AccountDetails.vue

```diff
@@ -16,13 +19,19 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import {
+  PropType,
+  defineComponent,
+  ref,
+  watch,
+} from 'vue';
 import { IonContent, IonPage } from '@ionic/vue';
 import { PROTOCOL_VIEW_ACCOUNT_DETAILS } from '@/constants';
 import { ROUTE_ACCOUNT_DETAILS, ROUTE_ACCOUNT_DETAILS_ASSETS } from '@/popup/router/routeNames';
 
 import AccountDetailsBase from '@/popup/components/AccountDetailsBase.vue';
 import AccountDetailsNavigation from '@/popup/components/AccountDetailsNavigation.vue';
+import { IonicLifecycleStatus } from '@/types';
```

Comment:
Please put this line before the constants import.

Reply:
Damn auto-imports :S